src/Isolation/Masa.BuildingBlocks.Isolation/MultiTenant/Tenant.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Masa.BuildingBlocks.Isolation.MultiTenant;
+public class Tenant
+{
+    public string Id { get; set; }
+
+    public Tenant(object id) : this(id.ToString() ?? String.Empty)
```

Comment:
id?.ToString() ?? throw exception xxx

Reply:
The original intention here is to allow the tenant to be manually set to be empty, and adjust the tenant id through code to achieve the purpose of changing the DbContext and skipping the tenant filtering. But after thinking about it carefully, I hope that changing the DbContext can be done by creating a new DbContext through IUnitOfWorkManager, and skipping tenant filtering can be done through IDataFilter. The design here is not suitable.